### DISCUSSION TITLE
What is "remnant untrusted"?

### DISCUSSION THREAD

**Discussion by opusforlife2** [endless-sky/endless-sky#10625] [Lore]
This is used for conversation branches in a couple of places, but there seems to be no place where it is actually set or removed. Is this future planned content? Or a leftover from removed content?

**Comment by TheGiraffe3** [comment]
There's definitely nowhere it's used.
A leftover from removed content, I'd guess?

**Reply by opusforlife2** [reply]
I forgot git blame was a thing. Zitchas added it two years ago in just 3 places, with no accompanying events that set or unset this boolean. So this looks like something incomplete that was planned on being added for the future.

**Reply by VaelophisNyx** [reply]
I'd bet it was to be used for if you help the deep get their hands on remnant tech and such
